test/grouping.jl

```diff
@@ -177,9 +202,7 @@ end
         # groupby() without groups sorting
         gd = groupby_checked(df, cols)
         @test names(parent(gd))[gd.cols] == colssym
-        @test sort(combine(identity, gd), colssym) ==
-            sort(combine(gd), colssym) ==
```

Comment:
Why not test `DataFrame(gd)` here and below? Can't hurt.

Reply:
I have added a test here (although it is not very clean unfortunately, because of the `combine` mechanics), but I agree it is good to have it here.